docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+# itk-version: 2.0.0
+version: "3"
+
+services:
+  phpfpm:
+    image: itkdev/php8.0-fpm
+    environment:
+      - XDEBUG_MODE=${XDEBUG_MODE:-off}
+      - XDEBUG_SESSION=${XDEBUG_SESSION:-0}
+      - PHP_MAX_EXECUTION_TIME=30
+      - PHP_MEMORY_LIMIT=256M
```

Comment:
Why the extra memory and the time is the same as the default value

Reply:
This is our default values for PHP containers?

Reply:
https://github.com/itk-dev/devops_itkdev-docker/blob/7da932868932b766079020171f071e34f758ac74/templates/symfony-6/docker-compose.yml#L29-L33